.travis.yml

```diff
@@ -3,46 +3,45 @@ sudo: required
 dist: xenial
 language: python
 python:
-    # python versions used in el7 SCL & supported fedora
-    - "3.6"
-    - "3.7"
+  - '3.6'
+  - '3.7'
 env:
   matrix:
     - DJANGO_MAX=2.2.100 DB=postgres TEST=pulp
     - DJANGO_MAX=2.2.100 TEST=docs
-  global:
-    secure: kzbI/DagIGIOFjVEyLyYGEi/xVwiq5M8FCEM3mOtXK9ocYs1Ky1ABkcgLrOLBb+S7SMNjy4XnUWGUxhBGWQLDw2/OimKFoHpCHGt1Cb4rXhWDSVPGgDXQzyW8pJsPKtAbemfx5a4oRP82vmpgB/htJ+8dE5CIAZWU8d9dbvQbro=
 matrix:
   fast_finish: true
 addons:
-    # postgres versions provided by el7 RHSCL (lowest supportable version)
-    postgresql: "9.6"
+  # postgres versions provided by el7 RHSCL (lowest supportable version)
+  postgresql: '9.6'
 services:
-    - postgresql
-    - redis-server
+  - postgresql
+  - redis-server
 install: source .travis/install.sh
 before_script: source .travis/before_script.sh
 script: source .travis/script.sh
-
 jobs:
   include:
-  - stage: deploy-pulpcore
-    script: bash .travis/deploy.sh pulpcore
-    if: tag =~ ^pulpcore-3.0*
-  - stage: deploy-plugin
-    script: bash .travis/deploy.sh plugin
-    if: tag =~ ^pulpcore-plugin*
-  - stage: publish-beta-docs
-    script: bash .travis/publish_docs.sh beta
-    env:
-      - DJANGO_MAX=2.2.100
-      - DB=postgres
-      - TEST=docs
-    if: tag =~ ^pulpcore-3.0*
-  - stage: publish-nightly-docs
-    script: bash .travis/publish_docs.sh nightly
-    env:
-      - DJANGO_MAX=2.2.100
-      - DB=postgres
-      - TEST=docs
-    if: type != pull_request
+    - stage: deploy
+      script: skip
+      deploy:
+        provider: pypi
+        distributions: sdist bdist_wheel
+        user: pulp
+        password:
+          secure: "ej2lekfOr8S3Du6D6HPlB20+g5jwgZz75ouHlqIpc+IAgdAxmpn5vkspsAevMtchLw7sNgDAAbl0bQVacMDnn+zbo21s93OXcOmYzgy4IlTgopc+xi1XaKFdLMLiEiZ+KOGVJgg7yQwS3yeS28FARZGNDYTF/+N7eITteDThcG4="
+      if: tag =~ ^pulpcore-3.0*
+    - stage: publish-beta-docs
+      script: bash .travis/publish_docs.sh beta
+      env:
+        - DJANGO_MAX=2.2.100
+        - DB=postgres
+        - TEST=docs
+      if: tag =~ ^pulpcore-3.0*
```

Comment:
maybe change this to if: tag IS present like here https://github.com/pulp/pulp_file/blob/master/.travis.yml#L25